tests/unit/amo/pages/TestUserProfileEdit.js

```diff
@@ -528,6 +556,41 @@ describe(__filename, () => {
     });
   });
 
+  it('captures input field changes for a reviewer', () => {
+    const fields = [
+      'biography',
+      'displayName',
+      'homepage',
+      'location',
+      'occupation',
+      'reviewerName',
```

Comment:
I suppose you could shorten this list a bit. I think you only need to test the `reviewerName` field.

Reply:
Admitedly it's me being a little paranoid, but I wanted to make sure that when a reviewer logs in, they can modify every field, not just reviewer name.

Reply:
This is covered by the other test case.